cvmfs/statistics.cc

```diff
@@ -136,10 +136,30 @@ string Statistics::PrintJSON() {
   return json_statistics.GenerateString();
 }
 
+/**
+ * Snapshot current state of the counters in a map of string and non-atomic
+ * CounterInfo struct.
+ * Elements will either be updated or inserted into the map.
+ * 
+ * Note: This function does NOT clear previous elements part of the map.
+*/
+void Statistics::SnapshotCounters(
+                        std::map<std::string, CounterInfo> *counters,
+                        uint64_t *monotonic_clock) {
+  MutexLockGuard lock_guard(lock_);
+  *monotonic_clock = platform_realtime_ns();
```

Comment:
Should it be monotonic time or real time? If monotonic time, please change to. Also, if ns precision is not needed, we can just use `platform_monotonic_time()`, or `time()` for realtime.
```suggestion
  *monotonic_clock = platform_monotonic_time_ns();
```

Reply:
no we want realtime. renamed to timestamp to mark when the snapshot of the counters was taken